assets/model_monitoring/components/src/data_drift_compute_metrics/compute_data_drift.py

```diff
@@ -41,6 +43,12 @@ def compute_data_drift_measures_tests(
     numerical_production_df = production_df.select(numerical_columns_names)
     categorical_production_df = production_df.select(categorical_columns_names)
 
+    if len(numerical_columns_names) == 0 and \
```

Comment:
If just return, would the following component fail?

Reply:
no its not failing, here is the link https://ml.azure.com/experiments/id/4b4e34ef-331f-4b1a-b1b0-88615d6c45ad/runs/plucky_cushion_912cr3jxzz?wsid=/subscriptions/1aefdc5e-3a7c-4d71-a9f9-f5d3b03be19a/resourcegroups/TestingTeamProjectRG/providers/Microsoft.MachineLearningServices/workspaces/shrsha-eastus-test&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#